src/medCore/database/medDatabaseModel.h

```diff
@@ -22,6 +22,8 @@ class medAbstractDatabaseItem;
 class medDatabaseModelPrivate;
 class medDataIndex;
 
+QString anonymise(const QString name);
```

Comment:
So it's a utility function ? :) It's just that it bothers me that it's not in a namespace anywhere (either in a class or a ```med ``` namespace or whatever), because any file that includes (directly or indirectly) medDatabaseModel sees ```anonymise``` as a global function belonging to nothing.
I won't insist though.